bzl/deps.bzl

```diff
@@ -0,0 +1,78 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def djinni_deps():
+
+    native.android_sdk_repository(name = "androidsdk")
```

Comment:
i would imagine this particular setting/dep would be separate from other project deps.

Reply:
Yeah I wasn't quite sure exactly how to model this. Maybe this lives in WORKSPACE and we don't consider it a dependency